packages/next/build/webpack/loaders/next-babel-loader.js

```diff
@@ -4,9 +4,37 @@ import babelLoader from 'babel-loader'
 
 // increment 'c' to invalidate cache
 const cacheKey = 'babel-cache-' + 'c' + '-'
+const nextBabelPreset = require('../../babel/preset')
+
+const getModernOptions = (babelOptions = {}) => {
+  const presetEnvOptions = Object.assign({}, babelOptions['preset-env'])
+  const transformRuntimeOptions = Object.assign(
+    {},
+    babelOptions['transform-runtime'],
+    { regenerator: false }
+  )
+
+  presetEnvOptions.targets = {
+    esmodules: true
+  }
+  presetEnvOptions.exclude = [
```

Comment:
This isn't needed right? Targets of `esmodules` implies these exclusions.

Not to mention, these exclusions are incorrect. They should be:
```
@babel/plugin-transform-regenerator
@babel/plugin-transform-async-to-generator
```